modules/sqs/main.tf

```diff
@@ -139,6 +151,88 @@ resource "aws_sqs_queue" "metadata_queue" {
   visibility_timeout_seconds  = 900 # Set to the lambda max time
 }
 
+## s3-inventory-queue - get_current_archive_list lambda reads from this queue to get what it needs to pull in next
+## ==================================================================================================================
+resource "aws_sqs_queue" "s3_inventory_queue" {
+  ## OPTIONAL
+  name                        = "${var.prefix}-orca-s3-inventory-queue.fifo"
+  fifo_queue                  = true
```

Comment:
Per our discussion on FIFO and how a FIFO DLQ we think will break the issue where processing would stop because order must be maintained, will leave as FIFO for now until we see some real world use/testing.